build/ci/goreleaser.yml

```diff
@@ -1,4 +1,4 @@
-project_name: "keboola_cli"
+project_name: "keboola-cli"
```

Comment:
`Alpine` Linux vyzaduje, aby bol nazov suboru a balicka rovnaky ... tj. `keboola-cli`. ... kaslem na to a bude to vsade s `-`, aj v `ZIP`.